test/lib/runtime/flow_test.exs

```diff
@@ -399,6 +399,15 @@ defmodule Ask.FlowTest do
     assert prompts == ["Do you smoke?", "Reply 1 for YES, 2 for NO"]
   end
 
+  describe "only step with skip logic null" do
```

Comment:
This test has never failed without introducing changes to `flow.exs`. By the rules of TDD this test must be deleted :)